src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java

```diff
@@ -69,6 +76,39 @@ void readBreakOnSchemaRegistryError(final Map<String, ?> configs) {
     }
   }
 
+  /**
+   * get reader schema from config if provided
```

Comment:
Can you change this comment to `parse reader schema...`

Reply:
Also, is parseReaderSchema a better name?

Reply:
Yess, parseReaderSchema makes more sense. Let me make that change.